### ISSUE TITLE
Add operation to engine.hpp

### ISSUE THREAD

**Issue by Mysterium-sch** [NVIDIA-AI-IOT/Lidar_AI_Solution#368]
I am trying to modify the code to export and run Voxelnext. I have add a custom node to handle the part of the forward method that are not directly supported. I need to add a custom push to the engine file. How would I add an operation?

  
  class FuseConvFeatures(torch.nn.Module):
      def forward(self, x_conv4, x_conv5, x_conv6):
          x_conv5.indices[:, 1:] *= 2
          x_conv6.indices[:, 1:] *= 4
  
          features = torch.cat([
              x_conv4.features,
              x_conv5.features,
              x_conv6.features
          ])
  
          indices = torch.cat([
              x_conv4.indices,
              x_conv5.indices,
              x_conv6.indices
          ])
  
          x_conv = x_conv4.replace_feature(features)
          x_conv.indices = indices
  
          features_cat = x_conv.features
          indices_cat = x_conv.indices[:, [0, 2, 3]]
          spatial_shape = x_conv.spatial_shape[1:]
  
          indices_unique, inv = torch.unique(indices_cat, dim=0, return_inverse=True)
  
          features_unique = features_cat.new_zeros(
              (indices_unique.shape[0], features_cat.shape[1])
          )
          features_unique.index_add_(0, inv, features_cat)
  
          x_out = spconv.SparseConvTensor(
              features=features_unique,
              indices=indices_unique,
              spatial_shape=spatial_shape,
              batch_size=x_conv.batch_size
          )
          return x_out
  
  @register_node(FuseConvFeatures.forward)
  def symbolic_fuse(self, ilayer, y, x_conv4, x_conv5, x_conv6):
      register_tensor(y)
  
      print(f"   --> FuseConv{ilayer} -> Output {get_tensor_id(y)}")
  
      nodes.append(
          helper.make_node(
              "FuseConvFeatures",
              [
                  get_tensor_id(x_conv4),
                  get_tensor_id(x_conv5),
                  get_tensor_id(x_conv6),
              ],
              [get_tensor_id(y)],
              f"fuse{ilayer}"
          )
      )
